src/opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -115,6 +115,19 @@ message Span {
     // Indicates that the span covers the client-side wrapper around an RPC or
     // other remote request.
     CLIENT = 2;
+
+    // Indicates that the span is used internally. Default value.
```

Comment:
Can we respect the order from Java?

Reply:
Updated. I assume we haven't released proto so it's safe to just change the order.